cmd/lotus-provider/migrate.go

```diff
@@ -161,7 +175,7 @@ environment variable LOTUS_WORKER_WINDOWPOST.
 	}
 
 	if !lo.Contains(titles, "base") {
-		_, err = db.Exec(ctx, "INSERT INTO harmony_config (title, config) VALUES ('base', '')", "base")
+		_, err = db.Exec(ctx, "INSERT INTO harmony_config (title, config) VALUES ('base', '')")
```

Comment:
We probably should have the migrator enable:
lpCfg.Subsystems.EnableWinningPost = true
And add a comment about diabling it in lotus-miner